app/common/proxyhandler.go

```diff
@@ -74,7 +65,7 @@ func ProxyHandler(w http.ResponseWriter, r *http.Request, path string, prefix st
 		req.Header.Add(OriginHostReqHeader, origin.Host)
 		req.URL.Scheme = HttpProtocol
 		req.URL.Host = origin.Host
-		req.URL.Path = proxyPath
```

Comment:
I know what you want, you want to call the API of all microservices through API gateway , but if you modify it this way will affect the previous code if user not using API gateway service, i did't test it whether it will cause other functions not work, my suggestion is that you should know how API gateway agent the http requests firstly, if you know it well, you will find this UI server based on the principle of API gateway, why? because there is no API gateway project when UI project was released, so , i think you should send email to me (huaqiaoz@vmware.com), i copy to my leader, then we can talk about it how to fix this issue. thanks